Spec/RealtimeClientPresence.swift

```diff
@@ -98,6 +98,86 @@ class RealtimeClientPresence: QuickSpec {
                 }
             }
 
+            // RTP11
+            context("get") {
+
+                context("query") {
+                    it("waitForSync should be true by default") {
+                        expect(ARTRealtimePresenceQuery().waitForSync).to(beTrue())
+                    }
+                }
+
+                // RTP11a
+                it("should return a list of current members on the channel") {
+                    let options = AblyTests.commonAppSetup()
+
+                    var disposable = [ARTRealtime]()
+                    defer {
+                        for clientItem in disposable {
+                            clientItem.close()
+                        }
+                    }
+
+                    let expectedData = "online"
+                    // Load 150 members (2 pages)
+                    waitUntil(timeout: testTimeout) { done in
+                        disposable += addRandomMembersToChannel("test", members: 150, data: expectedData, options: options) {
+                            done()
+                        }
+                    }
+
+                    let client = ARTRealtime(options: options)
+                    defer { client.close() }
+                    let channel = client.channels.get("test")
+
+                    var presenceQueryWasCreated = false
+                    ARTRealtimePresenceQuery.testSuite_injectIntoClassMethod("init") { // Default initialiser
+                        presenceQueryWasCreated = true
+                    }
+
+                    waitUntil(timeout: testTimeout) { done in
+                        channel.presence.get { membersPage, error in
+                            expect(error).to(beNil())
+
+                            let membersPage = membersPage!
+                            expect(membersPage.items).to(haveCount(100))
```

Comment:
BTW. I would not throw this away, this test is pretty much what is needed for RSP3 so perhaps just repurpose it and do both at the same time?